### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by alexsporn**

### CODE REVIEW

**Review by muXxer — APPROVED** [review]

File: documentation/docs/welcome.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Shimmer
+- Node Software
+- Welcome
+- explanation
+---
+
+# Welcome to INX-POI
+
+INX-POI enables generating and verifying Proof-of-Inclusion of blocks in the Tangle.
+Given a piece of data or transaction and the proof, one can verify whether it was included in the Tangle at a given point in time.
```
Comment (lucas-tortora) [inline documentation/docs/welcome.md]:
```suggestion
INX-POI enables you to generate and verify Proof-of-Inclusion of blocks in the Tangle.
Given a piece of data or transaction and the proof, you can verify whether it was included in the Tangle at any given time.
```

File: documentation/docs/welcome.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+description: INX-POI enables generating and verifying Proof-of-Inclusion of blocks in the Tangle.
```
Comment (lucas-tortora) [inline documentation/docs/welcome.md]:
```suggestion
description: INX-POI enables you to generate and verify Proof-of-Inclusion of blocks in the Tangle.
```

File: documentation/docs/welcome.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+## Setup
+
+The recommended setup is to use the provided [Docker images](https://hub.docker.com/r/iotaledger/inx-poi).
+These images are also used in our [HORNET recommended setup using Docker](http://wiki.iota.org/hornet/develop/how_tos/using_docker).
+
+## Configuration
+
+The extension is configured by default to connect to your HORNET instance.
+
+You can find all the configuration options in the [configuration section](configuration.md).
```
Comment (lucas-tortora) [inline documentation/docs/welcome.md]:
```suggestion
The extension will connect to your local HORNET instance by default.

You can find all the configuration options in the [configuration section](configuration.md).
```

File: documentation/docs/welcome.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## API
+
+The extension exposes a custom set of REST APIs that can be used to generate or validate proofs.
```
Comment (lucas-tortora) [inline documentation/docs/welcome.md]:
```suggestion
The extension exposes a custom set of REST APIs that you can use to generate or validate proofs.
```

**Review by lucas-tortora — APPROVED** [review]
Just some really minor comments. Awesome work! Thanks! :)

**Pull request merged**